package.json

```diff
@@ -22,6 +22,7 @@
   "license": "MIT",
   "homepage": "https://logux.io/",
   "repository": "logux/server",
+  "types": "./index.d.ts",
```

Comment:
Should I add it to Logux Client as well or TS can find `.d.ts` file automatically?

Reply:
TS will automatically find the file in most cases. I think this field can be deleted.

Reply:
Yeap, let’s delete it